lib/enigma.rb

```diff
@@ -37,4 +37,35 @@ def shifts(key, date)
     end
     shifts
   end
+
+  def to_indexes(string)
+    alphabet = Array("a".."z")<< " "
+    string_array = string.downcase.chars
+    string_array.map do |char|
+      alphabet.find_index(char)
+    end
+  end
+
+  def indexes_to_string(string, shifts)
+    alphabet = Array("a".."z")<< " "
+    shifts.map do |number|
+      alphabet.rotate(number)[0]
+    end.join
+  end
```

Comment:
`indexes_to_string` will take in an array of numbers, each number representing an index position in our `alphabet` array, then map over that numbers array rotating the `alphabet` array based on each number and returning the letter at the [0] index position. 
Once it has iterated over the entire numbers array we will call `join` on the new strings array to return a. single string.

Reply:
I'm just now realizing that having this method take both a string and an array of numbers as an argument is pointless and it should be refactored to only take the array of numbers as an argument